pkgs/data/fonts/comic-mono/default.nix

```
@@ -0,0 +1,35 @@
+{ lib, fetchFromGitHub }:
+
+let
+  version = "2020-12-28";
+in fetchFromGitHub {
+  name = "comic-mono-font-${version}";
+
+  owner = "dtinth";
+  repo = "comic-mono-font";
+  rev = "9a96d04cdd2919964169192e7d9de5012ef66de4";
+
+  postFetch = ''
+    mkdir -p $out/share/fonts
+    tar -z -f $downloadedFile --wildcards -x \*.ttf --one-top-level=$out/share/fonts
+
+    mkdir -p $out/etc/fonts/conf.d
+    cp ${./comic-mono-weight.conf} $out/etc/fonts/conf.d/30-comic-mono.conf
```

${} already copies it in the store, so we can link to save a few extra bytes

```suggestion
    ln -s ${./comic-mono-weight.conf} $out/etc/fonts/conf.d/30-comic-mono.conf
```

---

Thanks, fixed!